### DISCUSSION TITLE
A discussion to teach

### DISCUSSION THREAD

**Discussion by whippingdot** [CoderBoiis/Chips-game#11] [Ideas]
Hey, this is just a practice discussion. Remember that you use discussions for questions on the project, ideas for the project, such. This is a practice question.
## Intro to q
After finishing the python version of this project, we are going to move on to a website. We will use HTML, CSS, and JS for that. The website will be this same chips game, just better with a website. First we will finish this, and then we will move onto that. That will be a __real__ project. This is just our starter project to get things started on this game. We will do that for every game.
## Question
Comment down below answering my question, which is what are you good in. HTML, CSS, or JS. I will give a response to the question myself too.

**Comment by whippingdot** [comment]
HTML, and maybe JS

**Comment by NRafa1391** [comment]
JS
